src/vanilla/Model/CompositeTypeRb.cs

```diff
@@ -113,6 +114,18 @@ public virtual string ConstructModelMapper()
             return builder.ToString();
         }
 
+        public bool IsResourceGroupRequired()

```

Comment:
should this change be part of the Azure generator?

Reply:
Since the template is in vanilla flavor, then in order to compile, I need to add a virtual code here and override in Azure generator which is really not required for such a simple case. No change here.